### ISSUE TITLE
zpkdata does not work for SS

### ISSUE THREAD

**Issue by baggepinnen** [mfalt/Control.jl#17]
```julia
julia> sys = ss(P)
StateSpace:
A = 
          x1 
  x1   -1.0  
B = 
         u1 
  x1   1.0  
C = 
         x1 
  y1   1.0  
D = 
         u1 
  y1   0.0  

Continuous-time state-space model

julia> zpkdata(sys)
ERROR: MethodError: `tzero` has no method matching tzero(::Array{Float64,2}, ::Array{Float64,1}, ::Array{Float64,2}, ::Array{Float64,2})
Closest candidates are:
  tzero(::Array{Float64,2}, ::Array{Float64,2}, ::Array{Float64,2}, ::Array{Float64,2})
 in _zpk_kern at /local/home/fredrikb/.julia/v0.4/Control/src/analysis.jl:61
 in zpkdata at /local/home/fredrikb/.julia/v0.4/Control/src/analysis.jl:53

```

**Comment by mfalt** [comment]
Fixed in acb7ab1.

**Comment by baggepinnen** [comment]
Sweet, worked for me as well
